web/src/components/Board.res

```diff
@@ -16,35 +16,50 @@ module Column = {
   module Row = {
     module RView = {
       @react.component
-      let make = (~store: Store.t, ~changesAll: array<SearchTypes.change>, ~isChangeVisible) => {
+      let make = (
+        ~store: Store.t,
+        ~changesAll: array<SearchTypes.change>,
+        ~isChangeVisible: (HiddenChanges.changeStatus, MaskedChanges.changeStatus) => bool,
+      ) => {
         let (state, _) = store
+
         let (hiddenChanges, hiddenDispatchChange) = HiddenChanges.use(state.dexie, changesAll)
         let (pinnedChanges, pinnedDispatchChange) = PinnedChanges.use(state.dexie, changesAll)
+        let (maskedChanges, maskedDispatchChange) = MaskedChanges.use(state.dexie, changesAll)
         switch hiddenChanges->Belt.Array.length {
         | 0 => noChangeFound
         | _ =>
           hiddenChanges
-          ->Belt.Array.map(((hiddenStatus, change)) =>
-            isChangeVisible(hiddenStatus)
-              ? <Change.RowItem
-                  store
-                  key={change.url}
-                  change
-                  hiddenStatus
-                  hiddenDispatchChange
-                  pinnedStatus={PinnedChanges.simpleGetStatus(pinnedChanges, change)}
-                  pinnedDispatchChange
-                />
-              : React.null
-          )
+          ->Belt.Array.map(((hiddenStatus, change)) => {
+            let maskState = MaskedChanges.simpleGetStatus(maskedChanges, change)
```

Comment:
Perhaps rename "maskStatus"' to be consistent with "hiddenStatus" ?

Reply:
In this case, it is better not to rename it because the "hiddenStatus" variable is indeed the hidden status of a change.